### PR TITLE
User story 42/43 pd

### LINKED ISSUE #73: User Story 42, Merchant deactivates an item
As a merchant
When I visit my items page
I see a link or button to deactivate the item next to each item that is active
And I click on the "deactivate" button or link for an item
I am returned to my items page
I see a flash message indicating this item is no longer for sale
I see the item is now inactive

### LINKED ISSUE #74: User Story 43, Merchant activates an item
As a merchant
When I visit my items page
I see a link or button to activate the item next to each item that is inactive
And I click on the "activate" button or link for an item
I am returned to my items page
I see a flash message indicating this item is now available for sale
I see the item is now active

### PR DESCRIPTION
**Pull request by philjdelong**
## Description
Resolves #73, 
Resolves #74 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which maintains existing functionality)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## RSpec Results
```
Finished in 8.53 seconds (files took 2.44 seconds to load)
201 examples, 0 failures

Coverage report generated for RSpec to /Users/phildelong/turing/mod2/projects/monster_shop_part_1/coverage. 24
82 / 2482 LOC (100.0%) covered.
```

### CODE REVIEW

File: app/views/merchant/items/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<h1><%= link_to @merchant.name, "/merchants/#{@merchant.id}"%><span> Items</span></h1>
+<p align="center"><%= link_to "Add New Item", "/merchants/#{@merchant.id}/items/new" %></p>
```
Comment (DanielEFrampton) [inline app/views/merchant/items/index.html.erb]: Can you take out `align="center"` and instead give it an id and style that id in the CSS stylesheet?

**Review by DanielEFrampton — CHANGES_REQUESTED** [review]
Thanks for knocking this one out! One small change requested.

File: config/routes.rb [hunk 2]
```diff
@@ -54,6 +54,8 @@
   namespace :merchant do
     get '/', to: 'dashboard#index'
     get '/items', to: 'items#index'
+    patch '/items/:id/deactivate', to: 'items#update'
+    patch '/items/:id/activate', to: 'items#update'
```
Comment (mel-rob) [inline config/routes.rb]: We could make this even more efficient by combining into one path:

patch '/items/:id/activate_deactivate', to: 'items#update'

File: app/controllers/merchant/items_controller.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+class Merchant::ItemsController < Merchant::BaseController
+
+  def index
+    @merchant = Merchant.first
+  end
+
+  def update
+    item = Item.find(params[:id])
+    item.toggle!(:active?)
+    if item.active?
```
Comment (mel-rob) [inline app/controllers/merchant/items_controller.rb:10]: Here you could add conditional for the path, ie. if params[:activate_deactivate] == 'activate'

**Review by mel-rob — APPROVED** [review]

**Review by DanielEFrampton — APPROVED** [review]
Thanks!

**Pull request merged**